lab1.js

```diff
@@ -75,7 +76,27 @@ var sentence2 = 'Come over here so you can scratch my belly.';
  HINT: the 'split' method on String will be useful.
 */
 
-// your code goes here
+function splitString(stringToSplit, separator) {
+  var arrayOfStrings = stringToSplit.split(separator);
+  for (var i = 0; i < arrayOfStrings.length; i++) {
+    arrayOfStrings[i] = 'chirp';
+  }
+  // console.log(arrayOfStrings);
+}
+
+var space = ' ';
+
+splitString(sentence1, space);
+splitString(sentence2, space);
+
+function joinArray(arrayToJoin) {
+  var arrayOfStrings = arrayToJoin.toString(' ');
+  arrayOfStrings + '.';
+  console.log(arrayOfStrings);
+}
+
+joinArray(sentence1);
+joinArray(sentence2);
 
 assert(sentence1 === 'chirp chirp chirp.', 'sentence 1 should have 3 chirps');
```

Comment:
and finally, to get the asserts to pass, you'll need to somehow add a period to the end of sentence1 and sentence2.
In general, I'm glad to see you organizing your code using functions...that's an excellent habit to get into.